db/migrate/20140219183343_migrate_permissions.rb

```diff
@@ -22,34 +22,34 @@ def resource_type
     @resource_type ||= permissions.first.try(:resource_type)
   end
 
-  taxonomy_join_table = "taxable_taxonomies"
+  taxonomy_join_table = :taxable_taxonomies
   has_many taxonomy_join_table, :dependent => :destroy, :as => :taxable, :foreign_key => 'taxable_id'
-  has_many :locations,     :through => taxonomy_join_table, :source => :taxonomy,
-           :conditions => "taxonomies.type='Location'", :validate => false
-  has_many :organizations, :through => taxonomy_join_table, :source => :taxonomy,
-           :conditions => "taxonomies.type='Organization'", :validate => false
+  has_many :locations,     lambda{where("taxonomies.type='Location'")}, :through => taxonomy_join_table, :source => :taxonomy,
+           :validate => false
+  has_many :organizations, lambda{where("taxonomies.type='Organization'")}, :through => taxonomy_join_table, :source => :taxonomy,
```

Comment:
these don't appear correct, DB migrate's failing:

<pre>
00:03:30.886 wrong number of arguments (3 for 2)/var/lib/workspace/workspace/test_develop_pr_core/database/postgresql/ruby/1.9.3/app/models/concerns/has_many_common.rb:44:in `has_many'
00:03:30.888 /var/lib/workspace/workspace/test_develop_pr_core/database/postgresql/ruby/1.9.3/db/migrate/20140219183343_migrate_permissions.rb:27:in `<class:FakeFilter>'
00:03:30.888 /var/lib/workspace/workspace/test_develop_pr_core/database/postgresql/ruby/1.9.3/db/migrate/20140219183343_migrate_permissions.rb:11:in `<top (required)>'
</pre>